charts/fission-all/templates/pre-upgrade-checks/clusterrole.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Release.Name }}-fission-cr-preupgrade
+  annotations:
+    helm.sh/hook: pre-install
```

Comment:
Since deployment has pre-upgrade hook, we should have a pre-upgrade hook only here and across files in pre-upgrade-check folder.